Bottom: ca9c46877a96db574c5387990a9f437b760e32f3
Top:    9cdfa8fc338b5927493ad5770aa1d2d9bd7f4cc6
Author: Sean Hefty <sean.hefty@intel.com>
Date:   2012-07-26 16:18:08 -0700

Refresh of mpi-finalize

---

diff --git a/src/preload.c b/src/preload.c
index 026ff1c..52eaf1a 100644
--- a/src/preload.c
+++ b/src/preload.c
@@ -801,7 +801,6 @@ int select(int nfds, fd_set *readfds, fd_set *writefds,
 int shutdown(int socket, int how)
 {
 	int fd;
-	fflush(stdout);
 	return (fd_get(socket, &fd) == fd_rsocket) ?
 		rshutdown(fd, how) : real.shutdown(fd, how);
 }
